owa/app/js/components/manage_autogeneration_option/helpers.js

```diff
@@ -0,0 +1,69 @@
+import axios from 'axios';
+
+async function autogenerationOptionsHelper() {
+  const state = {};
+  const locations = await axios
+    .get('/openmrs-standalone/ws/rest/v1/location')
+    .then(
+      res => {
+        return res.data.results;
+      },
+      ({ response }) => {
+        console.log(response);
+      }
+    );
+  const genOptionAutogenerateSources = await axios
+    .get('/openmrs-standalone/ws/rest/v1/idgen/identifiersource')
```

Comment:
this will only work on your stand alone. use the apiHelper.js  it handles the url for you